File: apis-test/nestjs/auth.service.ts
// @ts-nocheck
import { Injectable } from '@nestjs/common';

@Injectable()
export class AuthService {
    async login(credentials: any) {
        return { token: 'mock-jwt-token', user: { id: 1, email: credentials.email } };
    }

    async signup(userData: any) {
        return { id: 2, ...userData };
    }
}

File: apis-test/dotnet/AuthService.cs
using System.Collections.Generic;

namespace MyApi.Services
{
    public class AuthService
    {
        public object Login(dynamic credentials)
        {
            return new 
            {
                Token = "mock-jwt-token",
                User = new { Id = 1, Email = credentials.email }
            };
        }

        public object Signup(dynamic userData)
        {
            return new { Id = 2, userData };
        }
    }
}
